packages/components/src/templates/next/components/internal/Askgov/AskgovWidget.tsx

```diff
@@ -0,0 +1,50 @@
+"use client"
+
+import { useEffect } from "react"
+
+import type { AskgovWidgetProps } from "~/interfaces"
+
+// Reference: https://github.com/opengovsg/askgov-widget
+export const AskgovWidget = ({
+  environment,
+  ...askgovProps
+}: AskgovWidgetProps) => {
+  const scriptUrl =
+    environment === "production"
+      ? "https://script.ask.gov.sg/widget.js"
+      : "https://script-staging.ask.gov.sg/widget.js"
+
+  const reloadAskgovScript = () => {
+    const scriptId = "isomer-askgov-script"
```

Comment:
[nit] no real issue, but prefer it to be outside of the component in `SCREAMING_SNAKE_CASE`